docker-compose.hobby.yml

```diff
@@ -32,9 +32,7 @@ services:
             - zookeeper
         volumes:
             - ./posthog/posthog/idl:/idl
-            - ./posthog/docker/clickhouse/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```

Comment:
This is a different mountpoint, destination is `/docker-entrypoint-initdb.d`

Reply:
Good point - sorry. I'm totally making work for you here.

I wonder if config.xml and users.xml should go into a `/posthog/docker/clickhouse/server` directory in the repo in that case and the compose file would contain:

```
- ./posthog/docker/clickhouse/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
- ./posthog/docker/clickhouse/server:/etc/clickhouse-server
```

Grepping the code base `docker-compose.dev.yml` would also need to be updated to match this.

The way it is now seems to have been there for 10 months now though, so I can't help but wonder what I've done wrong since no one else has seen this before. Or is just none of the dev done via docker.